<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Erenci</class>
 <widget class="QMainWindow" name="Erenci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>740</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SPQR eReNCi - a Remote Nao Console - LT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../usr/share/icons/default.kde4/16x16/categories/preferences-other.png</normaloff>../../../../../usr/share/icons/default.kde4/16x16/categories/preferences-other.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabControls">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>630</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>560</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabJointsControls">
       <attribute name="title">
        <string>Joints Controls</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="JointsController" name="jointsController" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>540</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>550</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAdvancedControls">
       <attribute name="title">
        <string>Advanced Controls</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="AdvancedControllers" name="advancedControls" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabVision">
       <attribute name="title">
        <string>Vision</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="Vision" name="visionController" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Connection</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="2">
           <widget class="QPushButton" name="btnConnect">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Connect</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+N</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btnDisconnect">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtConnect">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>27</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_24">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Nao Console IP:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="MotionMaker" name="motionMaker" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>134</width>
      <height>191</height>
     </rect>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveACopy"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>188</width>
      <height>164</height>
     </rect>
    </property>
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
    <addaction name="actionImportKME"/>
    <addaction name="actionReflectAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export To OpenRDK</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveACopy">
   <property name="text">
    <string>Save A Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import From OpenRDK</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionImportKME">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import From KME</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionReflectAll">
   <property name="text">
    <string>Reflect All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>JointsController</class>
   <extends>QWidget</extends>
   <header location="global">JointsController.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MotionMaker</class>
   <extends>QWidget</extends>
   <header location="global">MotionMaker.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Vision</class>
   <extends>QWidget</extends>
   <header location="global">Vision.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdvancedControllers</class>
   <extends>QWidget</extends>
   <header location="global">AdvancedControllers.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
